2022q3/wifi.adoc

```diff
@@ -0,0 +1,32 @@
+=== More wireless updates
+
+Links: +
+link:https://people.freebsd.org/~bz/wireless/[Bjoern's Wireless Work In Progress landing page] URL: link:https://people.freebsd.org/~bz/wireless/[https://people.freebsd.org/~bz/wireless/]
+link:https://wiki.freebsd.org/WiFi/Rtw88[Realtek rtw88 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Rtw88[https://wiki.freebsd.org/WiFi/Rtw88]
+link:https://wiki.freebsd.org/WiFi/Rtw89[Realtek rtw89 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Rtw89[https://wiki.freebsd.org/WiFi/Rtw89]
+link:https://wiki.freebsd.org/WiFi/Mt76[MediaTek mt76 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Mt76[https://wiki.freebsd.org/WiFi/Mt76]
+link:https://wiki.freebsd.org/WiFi/Ath11k[QCA ath11k status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Ath11k[https://wiki.freebsd.org/WiFi/Ath11k]
+
+Contact: Bjoern A. Zeeb <bz@FreeBSD.org>
+
+Currently development is mostly driven by Intel's iwlwifi driver again (see other report).
+As the saying goes ''each one helps the other'' so has work on Realtek's rtw89 driver helped to find a bug in LinuxKPI bothering iwlwifi users.
+For this status report the topic is mostly just on adding more drivers, which do need more LinuxKPI support.
+
+Various work in progress:
+
+ * Realtek's rtw88 PCI is in-tree as-is and after a fruitful discussion with Hans Petter at EuroBSDCon work on LinuxKPI USB support for the rtw88 USB WiFi dongles will continue.
+
+ * Realtek's rtw89 driver was committed to main but is not conected to the build yet.  Scanning already works but packets are not yet passing.  Having the driver in-tree already eased testing for users having that chipset in order to indentify more unimplemented LinuxKPI bits (some of which will help the other drivers as well) and reduced work for me.
+
+ * The next drivers to probably hit the tree will be based on MediaTek's mt76 driver (for 7921 and 7915) which I have compiling and started testing.
+
+ * Based on requests I am also working on ath11k to support STA mode given some vendors seem to ship Laptops with those chips.
+
+
+While some of this clearly benefits from work sponsored by the FreeBSD Foundation for iwlwifi and newer standard support, a lot of this is just free-time work.
```

Comment:
The FreeBSD Foundation

(uppercase T)

Oh, and thanks for all your free-time work!